core/base/module.php

```diff
@@ -329,6 +329,38 @@ protected function get_assets_relative_url() {
 		return 'assets/';
 	}
 
+	/**
+	 * Check if we should load the responsive css file.
+	 *
+	 * @param $widget_style_name
+	 * @return bool
+	 * @since 3.25.0
+	 * @access protected
+	 */
+	protected function should_load_responsive_css_file( $widget_style_name ): bool {
```

Comment:
@louiswol94 Can you please review this part specifically?

Reply:
@hein-obox How can 3rd parties that extend this base module make use of the same logic? Maybe add a filter here in the return statement so that they can hook in and search their own plugin folders for the CSS file. WDYT?

Reply:
@louiswol94 I have a follow up question: is there any documentation about get_frontend_file_url()?
I am not sure how this whole process works for 3rd parties.
In order for the get_frontend_file_url() to work, the 3rd party css files also need to be generated and moved to /wp-content/uploads/css/*.min.css. Is this part of the process supported?
In other words, is it possible for 3rd parties to use the custom breakpoints inside their scss files?

Reply:
@ramiy @rami-elementor What do you think about this? Do you know if custom breakpoints inside the 3rd party SCSS is supported?

Reply:
@louiswol94 Custom breakpoints inside SCSS are not supported by our product yet.